tests/ui_tests/test_configs.py

```diff
@@ -62,8 +66,28 @@ def _generate_fields():
     return fields
 
 
+def cluster_with_service(sdk_client: ADCMClient, path) -> Cluster:
+    """
+    Prepared cluster for test: create cluster, couple services and couple components.
+    """
+    bundle = sdk_client.upload_from_fs(path)
+    cluster_name = "_".join(path.split("/")[-1:] + [random_string()])
+    cluster = bundle.cluster_create(name=cluster_name)
+    return cluster
+
+
+@allure.step("Prepare cluster and get config")
+def prepare_cluster_and_config(sdk_client: ADCMClient, path, app):
+    """Upload bundle, create cluster and get config"""
+    cluster = cluster_with_service(sdk_client, path)
+    config = ClusterConfigPage(app.driver, app.adcm.url, cluster.cluster_id).open()
+    config.wait_page_is_opened()
+    return cluster, config
```

Comment:
Isn't it a duplicate? Either way return type is required + `wait_page_is_opened()` is implicitly called in `open` if I remember correctly

Reply:
in wait_page_is_opened() we also wait for progress_bar to hide

Reply:
hm, you're right. maybe change `open` to use `wait_page_is_opened`?
otherwise, we duplicate stuff for no reason, I guess